content/concepts/feathersjs.textile

```diff
@@ -29,7 +29,7 @@ What makes FeathersJS different from all the other JavaScript frameworks out the
 
 h2(#how-does-it-work). How does it work?
 
-At the heart of FeathersJS are services. Services are simply JavaScript objects or ES6 classes that represent an API endpoint. Each one implements a specific set of functions to perform CRUD operations on the data. To add on to services, Feathers offers hooks which act as middleware. These can be made to execute code before or after a service function is called, or when an error occurs within a service function. FeathersJS also sends out events, if set up, when CRUD operations are executed by services. When used with Socket.IO, WebSockets can be used to emit and receive these events allowing FeathersJS to communicate between client and server creating realtime functionality. Again, every part of Feathers is made with the goal of being easy to understand, maintain, apply - and above all adapt and add on to.
+At the heart of FeathersJS are services. Services are simply JavaScript objects or ES6 classes that represent an API endpoint. Each one implements a specific set of functions to perform CRUD operations on the data. To add on to services, Feathers offers hooks which act as middleware. These can be made to execute code before or after a service function is called, or when an error occurs within a service function. FeathersJS also sends out events, if set up, when CRUD operations are executed by services. When used with "Socket.IO":https://www.ably.io/concepts/socketio, WebSockets can be used to emit and receive these events allowing FeathersJS to communicate between client and server creating realtime functionality. Again, every part of Feathers is made with the goal of being easy to understand, maintain, apply - and above all adapt and add on to.
```

Comment:
It'd be good to make local references to things within docs, so `https://www.ably.io/concepts/socketio` => `concepts/socketio`

Reply:
fixed